config/sidekiq_scheduler.yml

```diff
@@ -120,7 +120,7 @@ VBADocuments::ReportUnsuccessfulSubmissions:
   description: "Weekly report of unsuccessful benefits intake submissions"
 
 VBADocuments::ReportMonthlySubmissions:
-  cron: "4 3 2 7 * * America/New_York"
+  cron: "4 3 2 15 * * America/New_York"
```

Comment:
I'm not familiar with this form of cron. Could you explain when you want this to run in plain English?

Reply:
this will run at 02:03:04 on the 15th day of each month.

Reply:
So that explains the first 5 (4 3 2 15 *). What's the 6th place (*) for?

Reply:
I believe the 5th position is for the month and the sixth is for the day of the week.